src/content/recipes/@vanilla-extract/css.md

```diff
@@ -0,0 +1,99 @@
+<div class="aside aside__no-top">
+
+This recipe assumes that you are using Storybook >= 7.0. If you haven't set up Storybook yet, run the following command in the root of your project:
+
+```shell
+npx storybook@latest init
+```
+
+</div>
+
+<RecipeHeader>
+
+How to setup Vanilla Extract and Storybook
+
+</RecipeHeader>
+
+Vanilla Extract Uses TypeScript as your CSS preprocessor. Allowing you to write type‑safe, locally scoped classes, variables and themes, then generate static CSS files at build time. Storybook is an industry-standard tool for developing and testing UI components in isolation.
+
+## Configure your Storybook builder
+
+<div class="aside aside__no-top">
+
+If you're already using Vanilla Extract in your project, you may already have these dependencies installed.
+
+</div>
+
+First of all, install the required dependencies:
+
+```shell
+# For Vite users
+yarn add -D @vanilla-extract/vite-plugin
+
+# For Webpack users
+yarn add -D @vanilla-extract/webpack-plugin mini-css-extract-plugin
+```
+
+### Vite
+
+For Vite users, Storybook uses your project's `vite.config.js` for it's build configuration. This means, if your project is using Vanilla Extract, it's already configured for your Storybook as well. 🎉
```

Comment:
@Integrayshaun wondering if you're ok with tweaking this sentence a bit, we could probably go with:
"
For Vite users, Storybook uses your project's `vite.config.js` for its build configuration. This means if your project uses Vanilla Extract, it's also already configured for your Storybook. 🎉
"